schema.sql

```diff
@@ -27,4 +27,19 @@ ALTER TABLE animals DROP COLUMN species;
 
 ALTER TABLE animals
 	ADD COLUMN species_id INTEGER REFERENCES species(id),
-	ADD COLUMN owner_id INTEGER REFERENCES owners(id);
\ No newline at end of file
+	ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
```

Comment:
- [ ] According to the  Project requirements. you should Create a table named vets with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
age: integer
date_of_graduation: date

- Kindly make this 👍🏻

Reply:
Sure I am adding the id as primary key. Thanks